packages/model-viewer/src/three-components/ARRenderer.ts

```diff
@@ -137,22 +143,34 @@ export class ARRenderer extends EventDispatcher {
       console.warn('Cannot present while a model is already presenting');
     }
 
-    scene.model.scale.set(1, 1, 1);
+    scene.setCamera(this.camera);
```

Comment:
There is a lot of AR-specific state that has to be tracked on a shared object (the `scene`). Is there a way for us to make these state changes more easily reversible? Maybe we need a different `scene` object specifically for AR?

Reply:
That's the way it was before; that had symmetric problems with not bringing state along, like lighting, hotspots, pivot. I think overall it's better this way, since we just make the changes in present() and $postSessionCleanup().